### PR TITLE
Adding Github Linguist

### PR DESCRIPTION
**Pull request by nkrabben**
This changes the necessary templates and data sources to add github linguist as a data source of formats.

### CODE REVIEW

File: _includes/fmt_list.html [hunk 1]
```diff
@@ -42,10 +42,10 @@ <h4 id="{{ fmt[0] | downcase }}">{{ fmt[0] }}</h4>
 <a href="http://apps.nationalarchives.gov.uk/PRONOM/{{ fmt[0] }}">{{ fmt[0] }}</a>
 &amp;
 <a href="https://github.com/digipres/digipres.github.io/blob/master/_sources/registries/pronom/{{ fmt[1]["source"] }}">{{ fmt[1]["source"] }}</a>
-{% elsif page.tool_id == "trid" %}
```
Comment (anjackson) [inline _includes/fmt_list.html]: Did you mean to remove this? Seems like this should still be present.

Comment (nkrabben) [inline _includes/fmt_list.html]: Yes, that's an overzealous delete on my part.

Comment (todrobbins) [inline _includes/fmt_list.html]: 👍🏻👍🏻

**Comment by nkrabben** [comment]
Reverted the commit on fddXML.zip and made a new commit to add tool_id==trim back.

**Comment by anjackson** [comment]
Great work, thanks!

**Pull request merged**